### ISSUE TITLE
How do I view training metric values for each iteration?

### ISSUE THREAD

**Issue by gabrielwomark** [Azure/mmlspark#953]
**Describe the bug**
Trying to build a ranking model using LightGBMRanker in pyspark but can't see any training metrics. I.e. even if I set `isProvideTrainingMetric` to `True` and `verbosity` to `2`, I can't see the `map` or `ndcg` on the training or validation sets. If I calculate those metrics by hand before and after training, the reranking after training causes a drop in the training metrics. 
If I have early stopping rounds set to 5, why is the model building 70 trees if `map` or `ndcg` are not going up?

** Details **
python3.6.9
spark 3.0.0
Building MML spark from source

**Comment by cadama** [comment]
I am having the same problem with `mmlspark_2.11-1.0.0-rc3` running on `spark 2.4.7` and `python 3.6`

**Comment by OscarDPan** [comment]
@cadama turned out the metrics are found in executor's stderr. I was using Yarn.

**Comment by cadama** [comment]
Thanks @OscarDPan. I still think is a bit unpractical to fetch them one by one from the executors. Maybe some logs aggregation would be nice here.

**Comment by ranadeepsingh** [comment]
Thank you for your question. When setting `isProvideTrainingMetric=True`, iteration metrics are written directly to the executor stderr logs in Spark cluster mode. Since this usage question has been clarified, we are closing this issue.
